package.json

```diff
@@ -11,6 +11,7 @@
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
     "dayjs": "^1.8.35",
     "global": "^4.4.0",
+    "humps": "^2.0.1",
```

Comment:
While using a library is nice, it does come at the cost of increased bundle/download size. As we're only doing a relatively simple string transformation, what do you think about just writing it ourselves? Or even better, just copy-pasting it from SO: https://stackoverflow.com/a/2970588/2071395 ;)

Reply:
The cost of the library is estimated at 1.5Kb: https://bundlephobia.com/result?p=humps@2.0.1

Reply:
We will also need the transformation back to snake case when passing back the data to the API. Would those two functions now equal 1.5kb? If not, I'm happy to just copy paste our own :)

Reply:
Right. They might actually. Let's go with the library for now, and we can always revisit later.